File: gdbm-on-python-2.6.4/skeleton.py
#!/usr/bin/python
# -*- coding: utf-8 -*-
# vi:encoding=utf-8

import os
import gdbm
from TestEnv import TestEnv

if not os.path.isdir( 'work'):
 os.mkdir( 'work')

try:
 os.remove( 'work/dbm_db')
except:
 pass

dbm_db= gdbm.open( 'work/dbm_db', 'c')

te= TestEnv()

te.checkComplainAndAdjustExpected( 0)

te.test( True)

te.checkComplainAndAdjustExpected( 1)

print te.result()

dbm_db.close()

# see also:

File: gcc-4.9/skeleton.c
#include <stdio.h>

#include "testenv.h"

int main( int argc, char** argv)
{
 
 checkComplainAndAdjustExpected( 0);

 test( 1);

 checkComplainAndAdjustExpected( 1);

 printf( "%s\n", result());

 return 0;
}
